src/rrt-viewer/MainWindow.cpp

```diff
@@ -54,6 +54,8 @@ MainWindow::MainWindow() {
 
     QLabel *stepSizeLabel = new QLabel("Step Size:");
 
+    QCheckBox *checkbox = new QCheckBox("Adaptive Stepsize Control", this);
```

Comment:
Could you rename this to ascCheckbox or something so it's a little more descriptive?